disperser/batcher/batcher.go

```diff
@@ -419,6 +422,16 @@ func (b *Batcher) observeBlobAge(stage string, batch *batch) {
 	}
 }
 
+func (b *Batcher) observeBlobAgeAndSize(stage string, batch *batch) {
+	for i, m := range batch.BlobMetadata {
+		requestTime := time.Unix(0, int64(m.RequestMetadata.RequestedAt))
+		b.Metrics.ObserveBlobAge(stage, float64(time.Since(requestTime).Milliseconds()))
+		for _, quorumInfo := range batch.BlobHeaders[i].QuorumInfos {
+			b.Metrics.IncrementBlobSize(stage, quorumInfo.QuorumID, int(m.RequestMetadata.BlobSize))
```

Comment:
do you want to record the original or encoded size?

Reply:
https://github.com/Layr-Labs/eigenda/blob/5692cacc062da330439f8e3ae6d8be3046becf98/disperser/common/blobstore/shared_storage.go#L110

Reply:
It's the original size